File: myHashSet.py
class MyHashSet(object):
    def __init__(self,n):
        self.array=[False]*n
    def add(self,key):
        self.array[key]=True
    def remove(self,key):
        self.array[key]=False
    def contains(self, key):
        return self.array[key]
obj=MyHashSet(10)
print(obj.array)
obj.add(4)
obj.add(3)
print(obj.array)
obj.remove(4)
print(obj.array)

print(obj.contains(3))

    

File: myHashSet.js
class MyHashSet{
    constructor(n){
        this.array= []
    }
    bool(n){
        for (let i=0; i<n;i++){
            this.array.push(false)
        }
    }
    add(key){
        for (let i=0;i<this.array.length;i++){
            if (i===key){
                this.array[i]= true
            }
        }
        return this
    }
    remove(key){
        for (let i=0; i<this.array.length; i++){
            if(i===key){
                this.array[i]=false
            }
        }
        return this 
    }
    contains(key){
        return this.array[key]
    }
}

obj=new MyHashSet()
obj.bool(5)

console.log(obj.add(3))

console.log(obj.contains(3))